### DISCUSSION TITLE
Get currenTime

### DISCUSSION THREAD

**Discussion by dromandev** [PierfrancescoSoffritti/android-youtube-player#1130] [General]
Hello everyone,

I'm currently using PierfrancescoSoffritti's YouTube library in my project, and I'm trying to implement a feature where I can get the current video time when a button is pressed. I've been exploring the library's documentation but couldn't find a clear method for achieving this.

Could someone please guide me on the correct approach to retrieve the current video time upon button press using this library? Any examples or code snippets demonstrating this functionality would be greatly appreciated.

Thank you in advance for your help!

**Comment by PierfrancescoSoffritti** [comment]
You could add a listener to the player and keep track of the current time in a variable.
